.github/workflows/cts.yml

```diff
@@ -1,11 +1,16 @@
 name: CTS
 
 on:
+  push:
+    branches-ignore: [
+        # We don't need to run on renovate PRs.
+        "renovate/**",
```

Comment:
It would be nice to run it automatically on the ones that update the CTS. Maybe we can give those a different name? Although I'd also question whether a special case here is really called for. 99% of the time it will be fine but every test we skip in CI creates a risk of things slipping through. If the compute cost of CI is getting too high then I think it would make sense to review the test list for CI and prioritize the things that give us the most coverage, in all CI runs.

Reply:
This is actually mis-documented. This prevents renovate PRs from triggering the _push_ trigger to prevent running twice per commit. They already run for the pull_request trigger.